Example/Podfile.lock

```diff
@@ -2,12 +2,12 @@ PODS:
   - CGFloatLiteral (0.2.0)
   - ManualLayout (1.3.0)
   - ReusableKit (1.1.0)
-  - RxCocoa (3.1.0):
-    - RxSwift (~> 3.1)
-  - RxKeyboard (0.4.0):
-    - RxCocoa (>= 3.0)
-    - RxSwift (>= 3.0)
-  - RxSwift (3.1.0)
+  - RxCocoa (3.5.0):
```

Comment:
Because without it the example won't compile. If you just run `pod install` it says it can't meet the requirements of the `3.4.0` that the library requires, while being locked to `3.1.0`.